src/ui/components/search/autocompletecomponent.js

```diff
@@ -188,8 +188,8 @@ export default class AutoCompleteComponent extends Component {
   }
 
   isQueryInputFocused () {
-    return document.activeElement &&
-      document.activeElement.className.includes(this._inputEl.substring(1));
+    return document.activeElement && document.activeElement.getAttribute('class') &&
```

Comment:
Looks good, Yen! I do have one suggestion which is to write this using optional chaining. Since we run this code through babel, the optional chaining should work on IE11. That version would look like this:  `return document.activeElement?.getAttribute('class')?.includes(this._inputEl.substring(1));`